app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -1,35 +1,78 @@
-ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope', 'networkRouterFormId', 'miqService', function($http, $scope, networkRouterFormId, miqService) {
+ManageIQ.angular.app.controller('networkRouterFormController', ['networkRouterFormId', 'miqService', 'API', function(networkRouterFormId, miqService, API) {
```

Comment:
Please add `$scope`. It's needed in on line 107 `$scope.angularForm.$setPristine(true);`

Reply:
Done